features/step_definitions/user_steps.rb

```diff
@@ -42,39 +42,44 @@
 Given(/^I am logged into the site$/) do
   visit new_user_session_path
   FactoryGirl.create(:user)
-  fill_in 'Email', with: 'newuser@sb.com' 
-  fill_in 'Password', with: 'password' 
+  fill_in 'Email', with: 'newuser@sb.com'
+  fill_in 'Password', with: 'password'
   click_on 'Log in'
 end
 
 When(/^I visit my own profile page$/) do
-  pending # express the regexp above with the code you wish you had
+ visit root_path
 end
 
 Then(/^I see my username$/) do
-  pending # express the regexp above with the code you wish you had
+  expect(page).to have_content("NewUser")
 end
 
 Then(/^I can see my avatar$/) do
-  pending # express the regexp above with the code you wish you had
+  visit page.find('#avatar')[:src]
+  page.status_code.should be 200
```

Comment:
Yea, so `should` is an older rspec syntax. You'll see it in code all over the internet.

The right way to do this would be:
````ruby
expect(page.status_code).to be 200
````

But this also isn't quite testing what you want to test. That code just tests that the page loads successfully. What you want is something like:

````ruby
expect(page).to have_css(<avatar css class>)
````

And fill in the css class there.